gooddata-sdk/docs/services/catalog-data-source.rst

```diff
@@ -228,6 +228,22 @@ The *gooddata_sdk.catalog_data_source* supports the following action API calls:
 
     Invalidate cache of your computed reports to force your analytics to be recomputed.
 
+* ``scan_data_source(data_source_id: str, scan_request: CatalogScanModelRequest = CatalogScanModelRequest(), report_warnings: bool = False)``
+
+    Returns *CatalogScanResultPdm*.
```

Comment:
Containing PDM and warnings.
Warnings contain information about columns, which were not added to the result PDM because their data types are not supported by GD.CN (e.g. ARRAY).